src/types/annotator.js

```diff
@@ -53,6 +53,16 @@
  * @prop {Target} target
  */
 
+/**
+ * Top and bottom positions of the bounding box created by the union of the
+ * highlight elements associated to an anchor.
+ *
+ * @typedef AnchorPosition
+ * @prop {string} tag - annotation tag
+ * @prop {number} top
```

Comment:
What coordinate system and units are `top` and `bottom` in?

Reply:
I added the unit (pixel) as a comment.